framework/src/Volo.Abp.AspNetCore.Mvc/Volo/Abp/AspNetCore/Mvc/Localization/AbpApplicationLocalizationScriptController.cs

```diff
@@ -53,14 +58,19 @@ await LocalizationAppService.GetAsync(input)
         );
     }
 
-    protected virtual string CreateScript(ApplicationLocalizationDto localizationDto)
+    protected virtual async Task<string> CreateScript(ApplicationLocalizationRequestDto input)
     {
+        var json = (await ScriptCache.GetAsync(nameof(AbpApplicationLocalizationScriptController)))?.Script;
+        if (json.IsNullOrWhiteSpace())
```

Comment:
While working with caches, we should always consider the scenario where the cache item is removed from cache for some reason. In this case, we should re-generate it (just like while we cached before)